internal/extractors/extract_data.go

```diff
@@ -9,13 +9,15 @@ import (
 // ExtractFunction is the type of the ExtractData function
 type ExtractFunction func(callback common.Callback, streamBatch ...StreamBatch)
 
+const channelBufferSize int = 1000
```

Comment:
Nit-picking here, but are re there any cache chunk related reasons to think that 1024 might be better?

Reply:
OK, it looks better :)